packages/manager/apps/pci-public-ip/src/pages/order/OrderPage.tsx

```diff
@@ -0,0 +1,692 @@
+import { useCallback, useContext, useEffect, useState } from 'react';
```

Comment:
since this file is starting to get huge, would it be possible to split it in multiple parts with OrderStep components ?